compiler/one-cmds/one-import-onnx

```diff
@@ -70,21 +70,21 @@ class TidyIONames:
         for idx in range(0, len(self.onnx_model.graph.input)):
             name = self.onnx_model.graph.input[idx].name
             if not name in self.initializers:
-                if '.' in name or ':' in name or name[:1].isdigit():
+                if '.' in name or ':' in name or '/' in name:
                     self.input_nodes.append(name)
                     name_alt = name.replace('.', '_')
                     name_alt = name_alt.replace(':', '_')
-                    if name_alt[:1].isdigit():
-                        name_alt = 'a_' + name_alt
+                    name_alt = name_alt.replace('/', '_')
+                    name_alt = 'a_' + name_alt  # prevent start with digit or _
```

Comment:
unconditionally add `a_` prefix to make it simple if there are multiple `_` in front of the name. so we don't need to check if name starts with numbers.